chore: Add Marketing Strategy (#1671)
asyncapi/community: annual-goals/2025_marketing_strategy.md

| **Contributing to AsyncAPI**                        | A guide to contributing to AsyncAPI, including coding standards, docs, design, GitHub usage, and community norms                            |          |
| **AsyncAPI Ask Me Anything (AMA) Sessions**                  | Open Q&A sessions with core maintainers and community leaders                                                                 |      

### Hackathons
To encourage hands-on learning and adoption of AsyncAPI, we will organize hackathons.

| **Challenge**                                       | **Description**                                                                                                               |     |
|-----------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|----------------------|
| **AsyncAPI Hackathon**                              | An AI hackathon focused on building innovative solutions using AsyncAPI data and existing technology.                                                           | 

## Communications

> ### Marketing Monthly Blog and Community Newsletter

Reviewer: ```suggestion
### Blog and Community Newsletter Communication
```

small wording improve.

it said `Marketing` and suggesting marketing blog is monthly, but later last bullet point says quater.

also problem is that text says

> Give appropriate support to the community manager:

but last bullet point is purely marketing related

# Community Marketing Strategy for AsyncAPI Initiative

> **AsyncAPI Initiative**
> 
> 
> **Community Marketing Strategy**

Reviewer: ```suggestion
```

I guess can be removed as the same is in the heading above

| **Goal 1: Encourage Adoption of AsyncAPI and EDA**| Inspire and motivate contributors worldwide to explore and utilize AsyncAPI specifications and event-driven architectures.                                                                        | - Create Comprehensive Tutorials: Develop detailed tutorials and guides that cover the basics to advanced concepts of AsyncAPI and EDA. <br> - Host Webinars and Workshops: Organize regular online sessions where experts explain AsyncAPI. <br> - Promoting the current docs we have and utilizing our mentor/maintainership program |
| **Expand Community Documentation**          | Enhance and expand the existing documentation to make it more comprehensive, accessible for contributors of all levels.                                                       | - Multilingual Support: Translate key documentation into multiple languages to make it accessible to a global audience. At least have one translation by the 2nd quarter of the year. <br> - User-Friendly Format: Ensure documentation is well-organized, searchable, and easy to navigate. |

> | **Utilize Community Initiatives and Mentorship Programs** | Enhance programs that encourage collaboration and mentorship within the community, supporting both new and experienced contributors.                                                                  | - Thematic Initiatives: Launch initiatives focusing on specific themes like security, performance, or new features. <br> - Support Groups: Promoting the existing working groups for ongoing projects and initiatives. |

Reviewer 2: The WG cleanup is not part of marketing, but CWG.

| **AsyncAPI Conference**                   | Collaborate with the conference coordination WG to organize a series of conferences around the world to promote AsyncAPI, share knowledge, and engage with the global community.                                                          | - Event Planning: Plan and schedule events in major countries around the world. <br> - Local Partnerships: Collaborate with local tech communities and organizations to host events. <br> - Speaker Lineup: Invite industry experts and community leaders to speak and share their knowledge. |
| **Promote Video Tutorials** |  Work with the community working group to create and promote high-quality video tutorials to help community members use and contribute to AsyncAPI projects and understand the EDA concepts effectively.                               | - Content Development: Work with experts in the community to create high-quality, in-depth video tutorials. This could be inform of the Bounty Program. <br> - Online Learning Platforms: Distribute the videos on our YouTube and other platforms. |

## Events and Activities

> ### Webinars and Workshops

Reviewer: just a comment that with such a large number of conferences, having webinars and being able to properly advertise them and get audience will be super hard. 

give it a thought if we really need to be so specific in the strategy document

like with hackathons -  it is enough to say we want to do one, we do not need to hardcode what about

same with webinars, I suggest that you could in general mention it, and first test one, try to advertise, find people willing to contribute, gain audience for the webinar - this will give you a feeling if this is a good direction or not - like you know, like MVP

especially that you have `Contributing to AsyncAPI` and we already have `Let's talk about contributing` (https://www.youtube.com/playlist?list=PLbi1gRlP7pigPBrBMaNQhUeniR1pdDMiY) - we just do not call it webinar, but event - but it is there, has a host - but never had a strong marketing support, upfront planning, technics to get people into the event (maybe we should transform it into the webinar, to have RSVP.

anyway, imho strategy does not have to be super precise on execution plan

## Demographics

We are a community of diverse people, and we have members across the globe.

## Purpose Statement

Building the Future of Event-Driven Architectures (EDA). Create open-source tools to easily build and maintain your event-driven architecture. All are powered by the AsyncAPI specification, the industry standard for defining asynchronous APIs.

## Marketing Goals and Objectives

The primary goal of our AsyncAPI community strategy is to foster the adoption and development of event-driven architectures (EDA) and asynchronous APIs. This initiative will serve as a platform to highlight exceptional talents within the global developer community and empower a passionate network of contributors. The objectives of the AsyncAPI community strategy are as follows:

- Drive initiatives to promote AsyncAPI and Increase adoption of EDA

> - Showcase Innovation and Promote Mentainership

Reviewer 2: ```suggestion
- Showcase Innovation and Promote Maintainership
```

- Organize webinars, workshops, and community challenges to provide hands-on experience.
- Expand outreach beyond Europe and Asia by engaging new regions through localized content, partnerships, and events.

| **Aim**                                    | **Description**                                                                                                                                                                                | **Tactics**                                                                                                                                                                                                                                           |
|---------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | **Goal 1: Encourage Adoption of AsyncAPI and EDA**| Inspire and motivate contributors worldwide to explore and utilize AsyncAPI specifications and event-driven architectures.                                                                        | - Create Comprehensive Tutorials: Develop detailed tutorials and guides that cover the basics to advanced concepts of AsyncAPI and EDA. <br> - Host Webinars and Workshops: Organize regular online sessions where experts explain AsyncAPI. <br> - Promoting the current docs we have and utilizing our mentor/maintainership program |

Reviewer 2: > Develop detailed tutorials and guides that cover the basics to advanced concepts of AsyncAPI and EDA.

Do you mean writing new tutorials and guides? And if so wouldn't it be a docs related issue instead of marketing?

| **Expand Community Documentation**          | Enhance and expand the existing documentation to make it more comprehensive, accessible for contributors of all levels.                                                       | - Multilingual Support: Translate key documentation into multiple languages to make it accessible to a global audience. At least have one translation by the 2nd quarter of the year. <br> - User-Friendly Format: Ensure documentation is well-organized, searchable, and easy to navigate. |

> | **Utilize Community Initiatives and Mentorship Programs** | Enhance programs that encourage collaboration and mentorship within the community, supporting both new and experienced contributors.                                                                  | - Mentorship Matchmaking <br> - Thematic Initiatives: Launch initiatives focusing on specific themes like security, performance, or new features. <br> - Support Groups: Promoting the existing working groups for ongoing projects and initiatives. |

Reviewer 2: What do you mean by Mentorship Matchmaking?

> | **Utilize Community Initiatives and Mentorship Programs** | Enhance programs that encourage collaboration and mentorship within the community, supporting both new and experienced contributors.                                                                  | - Mentorship Matchmaking <br> - Thematic Initiatives: Launch initiatives focusing on specific themes like security, performance, or new features. <br> - Support Groups: Promoting the existing working groups for ongoing projects and initiatives. |

Reviewer 2: We are no longer focusing on Mentorship but shifting the aspect to Maintainership

> | **Increase the Number of Community Maintainers** | Identify and train more maintainers to ensure the sustainability and continuous improvement of AsyncAPI projects. (Supporting the community effort)                                                                                | - Mentorship Program: Pair experienced maintainers with new contributors to guide them through the process of becoming maintainers through the Maintainership Program. <br> - Contributor Workshops: Conduct workshops focused on the responsibilities and tasks of a maintainer. <br> - Recognition and Rewards: Acknowledge and reward maintainers for their contributions to motivate ongoing participation. |

Reviewer 2: ```suggestion
| **Increase the Number of Community Maintainers** | Identify and train more maintainers to ensure the sustainability and continuous improvement of AsyncAPI projects. (Supporting the community effort)                                                                                | - Maintainership Program: Pair experienced maintainers with contributors to guide them through the process of becoming maintainers. <br> - Contributor Workshops: Conduct workshops focused on the responsibilities and tasks of a maintainer. <br> - Recognition and Rewards: Acknowledge and reward maintainers for their contributions to motivate ongoing participation. |
```

| **Goal 3:Drive Community Engagement**              | Promote a collaborative and inclusive environment that encourages active participation, knowledge sharing, and continuous learning among community members.                                      | - Interactive Platforms: Maintain active presence on the existing community platforms like Slack, GitHub, LinkedIn, Mastadon, YouTube, and Twitter to facilitate discussions. <br> - Regular Updates: Send out monthly newsletters  and blogs with updates, tips, and highlights to keep the community informed and engaged. <br> - Feedback Loops: Implement regular surveys and feedback sessions to understand community needs and improve engagement strategies. |

> | **AsyncAPI Conference**                   | Colaborate with the community manager to organize a series of conferences around the world to promote AsyncAPI, share knowledge, and engage with the global community.                                                          | - Event Planning: Plan and schedule events in major countries around the world. <br> - Local Partnerships: Collaborate with local tech communities and organizations to host events. <br> - Speaker Lineup: Invite industry experts and community leaders to speak and share their knowledge. |

Reviewer 2: ```suggestion
| **AsyncAPI Conference**                   | Collaborate with the conference coordination WG to organize a series of conferences around the world to promote AsyncAPI, share knowledge, and engage with the global community.                                                          | - Event Planning: Plan and schedule events in major countries around the world. <br> - Local Partnerships: Collaborate with local tech communities and organizations to host events. <br> - Speaker Lineup: Invite industry experts and community leaders to speak and share their knowledge. |
```

> | **Create a Library of High-Quality Video Tutorials and Courses** |  Work with the existing library to create high-quality video tutorials to help community members use and contribute to AsyncAPI projects and understand the EDA concepts effectively.                               | - Content Development: Work with experts in the community to create high-quality, in-depth video tutorials. This could be inform of the Bounty Program. <br> - Online Learning Platforms: Distribute the videos on our YouTube and other platforms. |

Reviewer 2: Is this the same as the Onboarding Videos that are currently being worked on under community, based on the training repo?

|-----------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|----------------------|
| **AsyncAPIConnect X (Twitter) Space/Podcast/LinkedIn Live**        | A webinar that provides an overview of AsyncAPI, its purpose, and how to get started                                          | Monthly              |
| **BuildBetter with AsyncAPI: Advanced AsyncAPI Usage**                         | Workshops focusing on more advanced usage and integrations of AsyncAPI                                                        | Monthly           |
| **Contributing to AsyncAPI**                        | A guide to contributing to AsyncAPI, including coding standards, docs, design, GitHub usage, and community norms                            | Quarterly            |
| **AsyncAPI Ask Me Anything (AMA) Sessions**                  | Open Q&A sessions with core maintainers and community leaders                                                                 | Quarterly       

## Communications

### Marketing Monthly Blog and Community Newsletter

> Give appropraite support to the community manager:

Reviewer 2: ```suggestion
Give appropriate support to the community manager:
```
